### PR TITLE
eth-contract-metadata@1.15.0

### PR DESCRIPTION
**Pull request by rekmarks**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -10059,11 +10059,16 @@ eth-block-tracker@^4.4.2:
...
-eth-contract-metadata@^1.11.0, eth-contract-metadata@^1.13.0:
+eth-contract-metadata@^1.11.0:
   version "1.13.0"
   resolved "https://registry.yarnpkg.com/eth-contract-metadata/-/eth-contract-metadata-1.13.0.tgz#9819d0e556ea2187da91d6b49ce96abc5fce2a73"
   integrity sha512-9CjXHX8IdXysUEvOHdbCsjdAwM1E98jaeK2HeOqm/9S/vOZ8YryaBBt/YSiBq3MkpCwf+d1pEQ53p96rsdy52w==
 
+eth-contract-metadata@^1.14.0:
+  version "1.14.0"
+  resolved "https://registry.yarnpkg.com/eth-contract-metadata/-/eth-contract-metadata-1.14.0.tgz#04f0ad1e6f6c49747808b4ed11012d665ca9953a"
+  integrity sha512-qDPzhBKGPW7ZeNqBVlo00v92b+pN3mGTuL72jgBtjjooGQWcyW0aWTuNGThOegEdZVJsAgJij+xP9t7pqO7kJQ==
```
Comment (whymarrh) [inline yarn.lock:10070]: We can de-dupe these:

```suggestion
eth-contract-metadata@^1.11.0, eth-contract-metadata@^1.14.0:
  version "1.14.0"
  resolved "https://registry.yarnpkg.com/eth-contract-metadata/-/eth-contract-metadata-1.14.0.tgz#04f0ad1e6f6c49747808b4ed11012d665ca9953a"
  integrity sha512-qDPzhBKGPW7ZeNqBVlo00v92b+pN3mGTuL72jgBtjjooGQWcyW0aWTuNGThOegEdZVJsAgJij+xP9t7pqO7kJQ==
```

**Review by whymarrh — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (rekmarks) [reply yarn.lock]: Hm, it wouldn't let me commit the suggestion, but: ca58a78

**Review by whymarrh — DISMISSED** [review]
LGTM!

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Review by whymarrh — APPROVED** [review]

**Pull request merged**